paper/paper.md

```diff
@@ -52,7 +50,7 @@ To leverage the RI in open-source identification workflows, a package providing
 # State of the field
 RI computation is contained in the most widely used GUI applications such as MS-DIAL [@Tsugawa2015] and MZmine2 [@Pluskal2010], the Galaxy tool metaMS [@Wehrens2014] and the python package CoreMS [@corilo2021].
 However, there is no standalone package which provides support for various computation methods based on homologous series (e.g alkanes [@Kovats1958], fatty acid methyl esters (FAMEs) [@Kind2009]) or the virtual carbon number [@Harangi2003].
-Additionally, existing tools expect input data in a fixed format and only perform RI computation and filtering inside the workflow run within the software.
+Additionally, existing tools expect input data in a fixed format and only perform RI computation and filtering inside the workflow runs within the software.

```

Comment:
> Additionally, existing tools expect input data in a fixed format and only perform RI computation and filtering inside the workflow runs within the software.

Additionally, existing tools expect input data in a fixed format and only perform RI computation as an intermediate step without reporting the actual values with the output.